nifi-bootstrap/src/main/java/org/apache/nifi/bootstrap/RunNiFi.java

```diff
@@ -175,7 +177,8 @@ private static void printUsage() {
         System.out.println("Status : Determine if there is a running instance of Apache NiFi");
         System.out.println("Dump : Write a Thread Dump to the file specified by [options], or to the log if no file is given");
         System.out.println("Diagnostics : Write diagnostic information to the file specified by [options], or to the log if no file is given. The --verbose flag may be provided as an option before " +
-            "the filename, which may result in additional diagnostic information being written.");
+                "the filename, which may result in additional diagnostic information being written.");
+        System.out.println("Status-history : Save the status history to the file specified by [options], or to the bootstrap log if no file is given");
```

Comment:
I would still think that describing the expected arguments up front (not after the first failure) would be useful